src/node.hpp

```diff
@@ -42,15 +43,17 @@ class Node {
     std::vector<std::reference_wrapper<Node>> previous;
     std::vector<std::reference_wrapper<Node>> next;
 
-    size_t finishedDependenciesCount = 0;
-
     // Blobs ready and waiting for execution
     std::unordered_map<session_key_t, std::unique_ptr<NodeSession>> nodeSessions;
-    BlobMap inputBlobs;
 
     // Input/Output name mapping and list of required inputs from previous nodes
     std::unordered_map<std::string, Aliases> blobNamesMapping;
 
+    // TODO make fields below const after integration of PipelineDefinition with Demultiplexer/Gather
```

Comment:
Why can't we do it before integration?

Reply:
Because it is scoped in another task. This requires changing all constructors of *Node clases which will be done in integration task, but with already correct values passed from config.